usb/usb_host_msc/include/usb/msc_host.h

```diff
@@ -142,8 +142,15 @@ __attribute__((deprecated("use API from esp_private/msc_scsi_bot.h")));
 /**
  * @brief Handle MSC HOST events.
  *
- * @param[in]  timeout_ms  Timeout in miliseconds
- * @return esp_err_t
+ * If MSC Host install was made with create_background_task=false configuration,
+ * application needs to handle USB Host events itself.
+ * Do not use if MSC host install was made with create_background_task=true configuration
+ *
+ * @param[in]  timeout  Timeout in FreeRTOS tick
+ * @return
+ *     - ESP_OK:          All events handled
+ *     - ESP_ERR_TIMEOUT: No events handled within the timeout
+ *     - ESP_FAIL:        Event handling finished, driver uninstalled. You do not have to call this function further
  */
 esp_err_t msc_host_handle_events(uint32_t timeout_ms);
```

Comment:
"Timeout in FreeRTOS tick" but argument name `timeout_ms`
Looks like an error, isn't it?